modular_skyrat/modules/stock_market/code/articles.dm

```diff
@@ -0,0 +1,125 @@
+/proc/consonant()
+	return pick("B","C","D","F","G","H","J","K","L","M","N","P","Q","R","S","T","V","W","X","Y","Z")
+
+/proc/vowel()
+	return pick("A", "E", "I", "O", "U")
+
+/proc/ucfirst(var/S)
+	return "[uppertext(ascii2text(text2ascii(S, 1)))][copytext(S, 2)]"
+
+/proc/ucfirsts(var/S)
+	var/list/L = splittext(S, " ")
+	var/list/M = list()
+	for (var/P in L)
+		M += ucfirst(P)
+	return jointext(M, " ")
+
+GLOBAL_LIST_EMPTY(FrozenAccounts)
+
+/proc/list_frozen()
+	for (var/A in GLOB.FrozenAccounts)
+		to_chat(usr, "[A]: [length(GLOB.FrozenAccounts[A])] borrows")
+
+/datum/article
+	var/headline = "Something big is happening"
+	var/subtitle = "Investors panic as stock market collapses"
+	var/article = "God, it's going to be fun to randomly generate this."
+	var/author = "P. Pubbie"
+	var/spacetime = ""
+	var/opinion = 0
+	var/ticks = 0
+	var/datum/stock/about = null
+	var/outlet = ""
+	var/static/list/outlets = list()
+	var/static/list/default_tokens = list( \
+		"buy" = list("buy!", "buy, buy, buy!", "get in now!", "ride the share value to the stars!"), \
+		"company" = list("company", "corporation", "conglomerate", "enterprise", "venture"), \
+		"complete" = list("complete", "total", "absolute", "incredible"), \
+		"country" = list("Space", "Argentina", "Hungary", "United States of America", "United Space", "Space Federation", "Nanotrasen", "The Wizard Federation", "United Kingdom", "Poland", "The Syndicate", "Australia", "Serbia", "The European Union", "The Illuminati", "The New World Order", "Eurasian Union", "Asian Union", "United Arab Emirates", "Arabian League", "United States of Africa", "Mars Federation", "Allied Colonies of Jupiter", "Saturn's Ring", "Fringe Republic of Formerly Planet Pluto"), \
```

Comment:
Better to replace stuff like Australia or the UAE with stuff from our backgrounds section

Reply:
Should be good I think